tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -57,7 +57,7 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
       val regex = "(.*).java".r
       val rustSwigExclude = List("InternalPointerMarker", "JNIReachabilityFence")
       // any class that we can't implement equals and hashCode for must be in this list
-      val iclExclude = List("AssociationType", "IronOxide")
+      val iclExclude = List("AssociationType", "IronOxide", "BlindIndexSearch")
```

Comment:
We may, someday, need to implement equals and hashcode for BlindIndexSearch if we want to reasonable support having multiple search indexing in an application at a time as they might get stored in a hashmap. I think this is okay for now, however.

Reply:
@clintfred Because it holds a RNG, we could do it disregarding that, but it felt weird.